src/Model/Transaction/CreditCardDetails.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace TeamGantt\Dues\Model\Transaction;
+
+use TeamGantt\Dues\Model\CreditCardType;
+
+class CreditCardDetails
+{
+    protected CreditCardType $cardType;
+
+    protected string $lastFour;
+
+    public function __construct(CreditCardType $cardType, string $lastFour)
+    {
+        $this->cardType = $cardType;
+        $this->lastFour = $lastFour;
+    }
+
+    public function getCardType(): CreditCardType
```

Comment:
Our use case will require something like:

```php
$transaction->getCreditCardDetails()->getCardType()->label
```

Might not be exactly this, since `getCreditCardDetails` is nullable in the event that the payment instrument is a paypal account. 

In order to get the human readable value of the enum.